backend/siarnaq/api/episodes/models.py

```diff
@@ -298,22 +299,28 @@ def initialize(self):
         populate the Challonge brackets, and create TournamentRounds.
         """
 
-        tournament_name_public = self.name_long
-        tournament_name_private = tournament_name_public + " (private)"
+        challonge_name_public = self.name_long
+        challonge_name_private = challonge_name_public + " (private)"
 
         # For security by obfuscation,
-        # and to allow easy regeneration of bracket
-        # In #549, use letters (even capitals!)
-        # and use more characters (altho staying under the 32-char lim).
-        key = random.randint(1000, 9999)
+        # and to allow easy regeneration of bracket,
+        # create a random string to append to private bracket.
+        # Note that name_short can be up to 32 chars
+        # while challonge_id_private has a 50-char limit
```

Comment:
Leaving comment here but haven't formed a strong opinion yet.

Does this belong here or in challonge module?